management-common/src/main/resources/application.yml

```diff
@@ -0,0 +1,5 @@
+management-web:
```

Comment:
I think it's a really good idea of externalizing this messages, however, are you really using them?

Reply:
Yes you are right the thing is I started out and figure out that at least in annotations I can reference because cause me an error "attibute must be a constant"(so I will check later), because it has to be revolved in compile time so I just forget to remove it, I'll put in my definition of done checklist